.github/workflows/build.yml

```diff
@@ -16,30 +16,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Python
-        uses: actions/setup-python@v1
+      - id: setup-python
+        uses: actions/setup-python@v2
         with:
-          python-version: 3.8
-      - name: Store installed Python version
-        run: |
-          echo "::set-env name=PY_VERSION::"\
-          "$(python -c "import platform;print(platform.python_version())")"
-      - name: Cache pip test requirements
-        uses: actions/cache@v1
-        with:
-          path: ${{ env.PIP_CACHE_DIR }}
-          key: "${{ runner.os }}-pip-test-py${{ env.PY_VERSION }}-\
-            ${{ hashFiles('**/requirements-test.txt') }}"
-          restore-keys: |
-            ${{ runner.os }}-pip-test-py${{ env.PY_VERSION }}-
-            ${{ runner.os }}-pip-test-
-            ${{ runner.os }}-pip-
-      - name: Cache pre-commit hooks
-        uses: actions/cache@v1
+          python-version: 3.9
+      - name: Cache linting environments
+        uses: actions/cache@v2
         with:
-          path: ${{ env.PRE_COMMIT_CACHE_DIR }}
-          key: "${{ runner.os }}-pre-commit-py${{ env.PY_VERSION }}-\
+          path: |
+            ${{ env.PIP_CACHE_DIR }}
+            ${{ env.PRE_COMMIT_CACHE_DIR }}
+          key: |
+            lint-${{ runner.os }}-\
+            py${{ steps.setup-python.outputs.python-version }}-\
+            ${{ hashFiles('**/requirements-test.txt') }}-\
+            ${{ hashFiles('**/requirements.txt') }}-\
             ${{ hashFiles('**/.pre-commit-config.yaml') }}"
```

Comment:
Remove this trailing quote.

Reply:
I think this change is coming in another skeleton update, but I made a suggestion below in case we want to do it here.

Reply:
@jsf9k As @hillaryj said, this should be resolved in https://github.com/cisagov/skeleton-generic/pull/62 once everyone's onboard with the changes. Do you want to go ahead and merge this as-is to then let a later Lineage 🐙 have the adjustment?

Reply:
Yes, I am fine with that.